webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx

```diff
@@ -250,42 +250,36 @@ const ColumnContainer = styled.div`
 `;
 
 const Container = styled(ColumnContainer)`
-    flex: 1;
-`;
-
-const MainWrapper = styled.div<{isRHSOpen: boolean}>`
-    flex: 1;
-    display: flex;
-    flex-direction: column;
-    max-width: ${({isRHSOpen}) => (isRHSOpen ? 'calc(100% - 400px)' : '100%')};
+    display: grid;
+    grid-auto-flow: column;
+    grid-auto-columns: 2fr minmax(400px, 1fr);
 
     @media screen and (min-width: 1600px) {
-        max-width: ${({isRHSOpen}) => (isRHSOpen ? 'calc(100% - 500px)' : '100%')};
+        grid-auto-columns: 2.5fr 500px;
     }
 `;
 
-const Main = styled.main<{isRHSOpen: boolean}>`
-    max-width: 780px;
-    width: min(780px, 100%);
-    padding: 20px;
-    flex: 1;
-    margin: 40px auto;
-    display: flex;
-    flex-direction: column;
+const MainWrapper = styled.div`
+    display: grid;
+    grid-auto-flow: row;
+    height: 100vh;
```

Comment:
Constraining to 100vh won't take the global header height into account.

```suggestion
    overflow-y: hidden;
```
`min-height: 0` would also work

the frustratingly simple reason that took me a long time to learn: https://datacadamia.com/web/css/grid/overflow#solution_set_an_explicit_minimum_not_to_auto